package.json

```diff
@@ -21,15 +21,15 @@
   "author": "Roi Avinoam",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/avinoamr/dbstream-mongo/issues"
+    "url": "https://github.com/panoplyio/dbstream-mongo/issues"
   },
-  "homepage": "https://github.com/avinoamr/dbstream-mongo",
+  "homepage": "https://github.com/panoplyio/dbstream-mongo",
   "devDependencies": {
     "mocha": "^1.21.0",
     "sift": "0.0.18"
   },
   "dependencies": {
-    "dbstream": "^1.0.1",
+    "dbstream": "https://github.com/panoplyio/dbstream#v1.0.3",
```

Comment:
Not sure what format is preferred, this or the one with `github:org/repo`, but have you noticed that in some places we add the `git+` prefix? Not sure what it does, but please take a look:

<img width="778" alt="Screen Shot 2022-08-31 at 6 45 21 PM" src="https://user-images.githubusercontent.com/43310906/187721601-b812c0d3-92d5-43fc-a435-dbe33bb17ca1.png">

Reply:
The [npm install docs](https://docs.npmjs.com/cli/v8/commands/npm-install) state that the prefix `git:` is used to identify node modules hosted in a git repo. 

Using only `https://` is working properly, I was able to install all and it was reflected well in the lock file. But, it seems that npm is "recommending" we specify it's a git repo. I'll update the deps to use `git+https:` and see if it's working properly locally. BTW the `github:` prefix is specific to github but seems to behave as the `git:` prefix.